composer.json

```diff
@@ -33,7 +33,6 @@
         "ext-libxml": "*",
         "ext-mbstring": "*",
         "ext-simplexml": "*",
-        "composer/package-versions-deprecated": "^1.8",
```

Comment:
not used by us was required for Sulu with PHP 7 support but not on 2.5 which min requirement is 8.0